app/src/main/res/values/colors.xml

```diff
@@ -50,4 +52,9 @@
     <!-- Debug Log -->
     <color name="terminal_green">#4AF626</color>
 
+    <!-- transaction detail -->
+    <color name="tx_detail_contact_name_label_text">#B2B2B2</color>
```

Comment:
`tx_details_` or if you're going to go for `tx_detail`, rename your activity class accordingly.

Reply:
Actually I'm not sure about the detail/details thing, but let's pick one and stick with it everywhere - detail**s** is only in your package name, so I think it's reasonable to go for `detail`.